playwright-tests/.env.example

```diff
@@ -0,0 +1,18 @@
+### Playwright test run .env configuration
+
+# Stand type for the wallet setup. Can be equal: testnet, mainnet
+STAND_TYPE=testnet
+
+# Link type for a test run. Can be equal: testnet, localhost
+STAND_LINK_TYPE=testnet
+
+# Wallet configurations
+## Secret phase of the wallet to set up the wallet extension
+WALLET_SECRET_PHRASE=
+## Wallet password
+WALLET_PASSWORD=QwerTY!123!
```

Comment:
I suggest leaving the environment variables empty and providing clear instructions for the required environment variables.

Reply:
fixed